framework/db/Connection.php

```diff
@@ -618,9 +625,10 @@ protected function initConnection()
      */
     public function createCommand($sql = null, $params = [])
     {
-        $command = new Command([
+        $command = \Yii::createObject([
```

Comment:
I think you should make a standard for this though, snippet from `yii\mail\BaseMailer.php:
````
protected function createMessage()
    {
        $config = $this->messageConfig;
        if (!array_key_exists('class', $config)) {
            $config['class'] = $this->messageClass;
        }
        $config['mailer'] = $this;
        return Yii::createObject($config);
    }
````

I'm not familiar enough with this to have a strong opinion, but regardless what you pick I think it should be consistent ;-)

What is the argument for `\Yii::createObject()` if we have `Instance::ensure()`?